humble_collection.yml

```diff
@@ -41,63 +36,39 @@ script:
       content: |
         #!/usr/bin/env sh
         set -e
+        export LUTRIS_SKIP_INIT=1
+        LUTRIS_CMD="lutris:rungame/humble-games-collection"
+        FIXED_SCHEME="$(echo "$1" | sed 's/?/\//')"
  
-        function cleanup() {
-                # Restore original config
-                echo "Restoring original config"
-                mv "${BACKUP_FILE_PATH}" "${CONFIG_FULL_PATH}"
-        }
-        IFS=,
-        read START_COMMAND LUTRIS_CONFIGS LUTRIS_DB_PATH <<< "$INPUT_LPATHS"
-
-        IFS="|"
-        read GAME_ID CONFIGPATH <<< $(sqlite3 "$LUTRIS_DB_PATH" \
-          'select id, configpath
-           from games
-           where installer_slug = "humble-games-collection-windows"
-           order by id asc
-           limit 1;')
-
-        # Find config file
-        CONFIG_FULL_PATH="${LUTRIS_CONFIGS}${CONFIGPATH}.yml"
-        if [ ! -f "$CONFIG_FULL_PATH" ]; then
-                echo "$CONFIG_FULL_PATH does not exist. Exiting."
-                exit 1
+        echo "$FIXED_SCHEME" > $GAMEDIR/drive_c/.auth
+        if command -v lutris 1&>/dev/null; then
+          lutris $LUTRIS_CMD
+        else
+          flatpak run net.lutris.Lutris $LUTRIS_CMD
```

Comment:
I believe you can use `flatpak info net.lutris.Lutris >/dev/null` to check if Lutris is installed via flatpak. Not sure if there's another/better way.

Reply:
I just didn't check for flatpak because I had already checked for the lutris binary. If you didn't have both then this obviously wouldn't work anyway, so I don't see a point in another check 🤷